src/panels/config/integrations/integration-panels/zha/dialog-zha-reconfigure-device.ts

```diff
@@ -56,53 +89,194 @@ class DialogZHAReconfigureDevice extends LitElement {
+                                    <div class="grid-item">
+                                      ${this.hass.localize(
+                                        `ui.dialogs.zha_reconfigure_device.attribute`
+                                      )}
+                                    </div>
+                                    <div class="grid-item">
+                                      <div>
+                                        ${this.hass.localize(
+                                          `ui.dialogs.zha_reconfigure_device.min_max_change`
+                                        )}
+                                      </div>
+                                    </div>
+                                    ${Array.from(
+                                      clusterStatus.attributes.values()
+                                    ).map(
+                                      (attribute) => html`
+                                        <span class="grid-item">
+                                          ${attribute.name}:
+                                          ${attribute.success
+                                            ? html`
+                                                <ha-svg-icon
+                                                  .path=${mdiCheckCircle}
+                                                ></ha-svg-icon>
+                                              `
+                                            : html`
+                                                <ha-svg-icon
+                                                  .path=${mdiExclamationThick}
+                                                ></ha-svg-icon>
+                                              `}
+                                        </span>
+                                        <div class="grid-item">
+                                          ${attribute.min}/${attribute.max}/${attribute.change}
+                                        </div>
+                                      `
+                                    )}
+                                  </div>
+                                `
+                              : ""}
+                          </div>
+                        `
+                      )}
+                    `
+                  : ""}
+              </div>
+            `
+          : ""}
       </ha-dialog>
     `;
   }
 
-  private _handleMessage(message: any): void {
-    if (message.type === LOG_OUTPUT) {
-      this._formattedEvents += message.log_entry.message + "\n";
-      const paperTextArea = this.shadowRoot!.querySelector("paper-textarea");
-      if (paperTextArea) {
-        const textArea = (paperTextArea.inputElement as IronAutogrowTextareaElement)
-          .textarea;
-        textArea.scrollTop = textArea.scrollHeight;
+  private _handleMessage(message: ClusterConfigurationEvent): void {
+    // this is currently here to hack rerendering because map updates aren't triggering rendering?
```

Comment:
You will need to make a shallow copy of the map to make it trigger rendering:
`this._clusterConfigurationStatuses = new Map(this._clusterConfigurationStatuses)`
Or call `this.requestUpdate()`